src/app/Providers/components/ProvidersTable/VMware/__tests__/VMwareProvidersTable.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
```

Comment:
IMO we should name this file with the extension `.test.tsx`, it's convenient to have the test file and the component file named differently so you can tell them apart in the tabs of your editor.

Reply:
I agree as I experience confusion while having the component file opened at same time.